rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt

```diff
@@ -580,11 +588,12 @@ open class Kotlin11Generator(
     }
 
     //only for toplevel Exts
-    protected fun PrettyPrinter.createMethodTrait(decl: Toplevel) {
-        if (decl.isExtension) return
+    protected fun PrettyPrinter.createModelMethodTrait(decl: Toplevel) {
+        if (!decl.isToplevelExtension) return
 
         + "@JvmStatic"
-        block("fun create(lifetime: Lifetime, protocol: IProtocol): ${decl.name} ") {
+        + "@JvmName(\"internalCreateModel\")"
```

Comment:
`internal` methods in Kotlin are compiled in such a way that they are `public` in JVM but their name is mangled. I want to ensure that this method is not callable from Java or Kotlin outside of RD (since this is an internal function) but to allow it to be called via reflection for some specific purposes (like `ProtocolExtListenerManager`). This combination of `internal` and specific JVM name allows me to do this.

Reply:
but this method is not `internal`

Reply:
Oh, okay my bad